notebooks/depth_time_series.ipynb

```diff
@@ -0,0 +1,2310 @@
+{
```

Comment:
<p>We need to address the issue of sampling bias here. Are there <em>actually</em> less tuna, or are tuna being measured less? This is a hard question with no one perfect answer, but it is the question all researchers will be asking at this point in the analysis. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/iobis/pyobis/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>I will think that tuna is being measured less because this particular species is categorized as "Near Threatened" which means it does have more significant numbers than we can see on this plot. I do not know why there was a spike there initially around 1966 when the other records are much nearer to the surface. I found that occurrence records with <code>depth</code> for any species is very very less, and I think this might be because most of the surveys for species are aerial surveys. Should I leave an open question at this point in the analysis?</p>